otter/worker/launch_server_v1.py

```diff
@@ -308,6 +308,13 @@ def _check_server_created(f):
         d.addBoth(_check_results, f)
         return d
 
+    def _create_with_delay():
+        d = _treq.post(path, headers=headers(auth_token),
+                       data=json.dumps({'server': server_config}), log=log)
+        # Add 1 second delay to space 1 second between server creations
+        d.addCallback(delay, clock, 1)
```

Comment:
That does sound much cleaner way to do it. It makes its intent very clear. I got stuck in current code base and thought from that angle :). As you suggested, this doesn't need to block this PR now. Instead the idea can be utilized in converger.